go.mod

```diff
@@ -193,10 +191,10 @@ require (
 	gotest.tools v2.2.0+incompatible
 	k8s.io/api v0.21.1
 	k8s.io/apimachinery v0.21.1
-	k8s.io/apiserver v0.20.5
+	k8s.io/apiserver v0.20.6
```

Comment:
if you can update to `v0.20.7` that contains a vuln fix, which will make vulns scan happy even it is a false positive

Reply:
This and the other one you mentioned are overwritten below in the `replace` call together with all the other kubernetes dependencies.
Let's leave this for a future PR as discussed offline.